courses/free-courses-hi.md

```diff
@@ -306,6 +306,7 @@
 ### Networking
 
 * [3.4 Computer Networks (Complete Playlist)](https://www.youtube.com/playlist?list=PLmXKhU9FNesSjFbXSZGF8JF_4LVwwofCd) - Knowledge Gate
+* [Computer Networking (in one video)](https://youtu.be/IPvYjXCsTg8) - Kunal Kushwaha
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
No shortened links!